CHANGELOG.md

```diff
@@ -158,5 +158,6 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - Fixed a bug in which `nn.models.GAT` did not produce `out_channels`-many output channels ([#4299](https://github.com/pyg-team/pytorch_geometric/pull/4299))
 - Fixed mini-batching with empty lists as attributes ([#4293](https://github.com/pyg-team/pytorch_geometric/pull/4293))
 - Fixed a bug in which `GCNConv` could not be combined with `to_hetero` on heterogeneous graphs with one node type ([#4279](https://github.com/pyg-team/pytorch_geometric/pull/4279))
+- Fixed a bug in which the custom activation functions in GraphGym did not work. ([#5243](https://github.com/pyg-team/pytorch_geometric/pull/5243))
```

Comment:
```suggestion
- Fixed a bug activation initialization in `custom_graphgym` examples. ([#5243](https://github.com/pyg-team/pytorch_geometric/pull/5243))
```
Just a thought - as the bug is not actually in graphgym, just that specific example